aries_cloudagent/messaging/credential_definitions/routes.py

```diff
@@ -61,7 +59,7 @@ class CredentialDefinitionSchema(Schema):
     )
     tag = fields.Str(
         description="Tag within credential definition identifier",
-        example=INDY_CRED_DEF_ID["example"].split(":")[-1]
+        example=INDY_CRED_DEF_ID["example"].split(":")[-1],
```

Comment:
What is our style policy on trailing commas? I never would have put one in until I saw them regularly in your code. Now I put them in and you take them out. Sing ho for the life :-/